### ISSUE TITLE
Error: invalid authentication config: open /run/kube-apiserver/token-auth-file: no such file or directory

### ISSUE THREAD

**Issue by schu** [kinvolk/lokomotive#347]
lokoctl version `v0.1.0-169-gef479dc9` (HEAD of master today).

Following the quickstart guide for AWS doesn't lead to a working cluster; the apiserver is not running:

```
[...]
Flag --insecure-port has been deprecated, This flag will be removed in a future version.
I0425 12:13:32.553216       1 server.go:656] external host was not specified, using 10.0.14.30
I0425 12:13:32.553634       1 server.go:153] Version: v1.18.2
Error: invalid authentication config: open /run/kube-apiserver/token-auth-file: no such file or directory
Usage:
  kube-apiserver [flags]
[...]
```

Possibly a regression from c0d340374a4bc0a92cf1a00ba3adb489d9b85ce9

/cc @invidian

**Comment by rata** [comment]
This also means a bug in the CI, that should be running for AWS. I **guess** that checking that the cluster is up is done by the CI, but maybe I'm wrong. If not, we should add a test there too for all platforms.

**Comment by schu** [comment]
I'm guessing the problem is that the `/run/kube-apiserver` directory doesn't exist. I don't see yet what is different in the CI setup for the directory to exist.

**Comment by schu** [comment]
```
      {{- if eq (int .Values.apiserver.replicas) 1 }}
        - name: data
          mountPath: /run/kube-apiserver
```

The directory is only mounted `if eq (int .Values.apiserver.replicas) 1`, i.e. it only works with setting `controller_count = 1` in the lokocfg (as done in the CI setup).

**Comment by invidian** [comment]
>This also means a bug in the CI, that should be running for AWS. I guess that checking that the cluster is up is done by the CI, but maybe I'm wrong. If not, we should add a test there too for all platforms.

The problem is, that we don't set HA setup in the CI and this is what was failing. Perhaps we should.

Thanks for reporting @schu. PR with the fix is opened: #353.
